app/uk/gov/hmrc/excisemovementcontrolsystemapi/controllers/PreValidateTraderController.scala

```diff
@@ -36,7 +37,8 @@ class PreValidateTraderController @Inject() (
   parseJsonAction: ParseJsonAction,
   preValidateTraderService: PreValidateTraderService,
   cc: ControllerComponents,
-  appConfig: AppConfig
+  appConfig: AppConfig,
+  correlationIdService: CorrelationIdService
```

Comment:
In the PreValidateTraderConnector it still generates a new correlationId, is that correct?